packages/taquito-rpc/test/taquito-rpc.spec.ts

```diff
@@ -2585,6 +2589,76 @@ describe('RpcClient test', () => {
       expect(content.metadata.operation_result.level).toEqual(0);
       done();
     });
+
+    it('should access the properties of the operation type tx_rollup_dispatch_tickets, proto13', async (done) => {
+      httpBackend.createRequest.mockReturnValue(Promise.resolve(blockJakartanetSample));
+
+      const response = await client.getBlock();
+      const content = response.operations[3][0]
+        .contents[4] as OperationContentsAndResultTxRollupDispatchTickets;
+
+      expect(content.kind).toEqual(OpKind.TX_ROLLUP_DISPATCH_TICKETS);
+      expect(content.source).toEqual('tz1inuxjXxKhd9e4b97N1Wgz7DwmZSxFcDpM');
+      expect(content.fee).toEqual('835');
+      expect(content.counter).toEqual('252405');
+      expect(content.gas_limit).toEqual('4354');
+      expect(content.storage_limit).toEqual('86');
+      expect(content.tx_rollup).toEqual('txr1YMZxstAHqQ9V313sYjLBCHBXsvSmDZuTs');
+      expect(content.level).toEqual(4);
+      expect(content.context_hash).toEqual('CoV7iqRirVx7sZa5TAK9ymoEJBrW6z4hwwrzMhz6YLeHYXrQwRWG');
+      expect(content.message_index).toEqual(0);
+      expect(content.message_result_path).toBeDefined();
+      expect(content.message_result_path![0]).toEqual(
```

Comment:
I don't think that the Non-Null Assertion Operator `!` is needed, because `message_result_path` cannot be undefined in the `OperationContentsAndResultTxRollupDispatchTickets` interface.